iot-hub/Samples/device/ImportExportDevicesSample/Program.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.RetryPolicies;
+
+namespace ImportExportDevices
+{
+    public class Program
+    {
+        //user variables -- add a config file and put these in settings? 
+
+        //IoT Hub connection string. You can get this from the portal.
+        // Log into https://azure.portal.com, go to Resources, find your hub and select it.
+        // Then look for Shared Access Policies and select it. 
+        // Then select IoThubowner and copy one of the connection strings.
+        public static string IoTHubConnectionString = "<connection string to an existing IoT hub>";
+
+        // When copying data from one hub to another, this is the connection string
+        //   to the destination hub, i.e. the new one.
+        public static string DestIoTHubConnectionString = "<connection string to a IoT hub to copy devices to>";
+
+        // Connection string to the storage account used to hold the imported or exported data.
+        // Log into https://azure.portal.com, go to Resources, find your storage account and select it.
+        // Select Access Keys and copy one of the connection strings.
+        public static string storageAccountConnectionString = "<your storage account connection string>";
+
+        // Container used to hold the blob containing the list of import/export files.
+        // This is a module-wide variable.
+        public static CloudBlobContainer cloudBlobContainer;
+
+        // Name of blob container holding the work data.
+        public static string containerName = "devicefiles";
+
+        // Name of the file used for exports and imports. 
+        // This is set by the IoT SDK, and can't be changed.
+        public static string deviceListFile = "devices.txt";
+
+        public static void Main(string[] args)
+        {
+            //To use this sample, uncomment the bits you want to see run.
+
+            //The size of the hub you are using should be able to manage the number of devices 
+            //  you want to create and test with.
+            //For example, if you want to create a million devices, don't use a hub with a Basic sku.
+
+            // You must run this; it creates the blob storage resource.
+            string containerURI = PrepareStorageForImportExport();
+
+            //Console.WriteLine("Create new devices for the hub.");
```

Comment:
Please remove commented code.

Reply:
Not all of the code is to be run every time the customer runs it. He gets to pick which bits he runs. It's a multi-use sample.

Reply:
In that case please create two scenarios. Follow the coding patterns observed in other samples in the same folder: a single Program.cs and a class file that you can directly reference in any other application (e.g. UWP or Xamarin, etc).